cms/core/blocks/section_blocks.py

```diff
@@ -41,15 +43,27 @@ class Meta:
         block_counts: ClassVar[dict[str, dict]] = {"related_links": {"max_num": 1}}
 
 
+class GlossarySectionBlock(StructBlock):
+    title = HeadingBlock(default="Definitions")
```

Comment:
```suggestion
    title = HeadingBlock(default=_("Definitions"))
```

since this is a string that will be displayed in the front-end

Reply:
Done in [b8bb7a7](https://github.com/ONSdigital/dis-wagtail/pull/110/commits/b8bb7a7c6b099cac41b09b2e205ba7b25d7c240c)

Reply:
Note that since we won't be using the dedicated glossary section, the pre-population of the heading block field won't be implemented.